extensions/amp-font/amp-font.md

```diff
@@ -82,19 +82,19 @@ Time in milliseconds after which the we don't wait for the custom font to be ava
 
 ##### on-load-add-class
 
-CSS class that would be added to the `document.documentElement`  after making sure that the custom font is available for display. This attribute is optional.
+CSS class that would be added to the `document.body` after making sure that the custom font is available for display. This attribute is optional.
```

Comment:
because of `fixed` layer and backward compatibility, let's keep the "main" case still `document.documentElement` and add to/document `body` only for shadow-dom case.

Reply:
I'll keep the docs with `document.body` to indicate that that is our preference going forward and change the code to add the classes to both `document.documentElement` and `document.body` for backward compatibility. Will that work?

Reply:
still worried about fixed layer ( in iOS/embed mode ). the classes that are getting added here to body, are not getting added to the "second" `body` of fixed layer (not 100% sure but I believe that's the case since fixed layer is created before these events occur, I might be wrong) so we should not recommend folks doing `body.myclass`.